### ISSUE TITLE
When using foreshadowing, display the number of votes the bots start with.

### ISSUE THREAD

**Issue by haralkar** [sheibeck/pendulumbot#4]
This makes it more efficient to know if going for that extra vote might make a difference.

**Comment by sheibeck** [comment]
Are you saying, display the vote total up at the top of the app? Or am I missing something about the number of starting votes for bots?

**Comment by haralkar** [comment]
Yes, display the total votes would be best, then you can see at a glance how much effort you need to put into "politics", for your votes to be enough to have privilege.
